pkg/shp/reader.go

```diff
@@ -19,17 +17,18 @@ type Reader struct {
 	bbox         Box
 	err          error
 
-	shp        io.ReadSeeker
-	shape      Shape
-	num        int32
-	filename   string
+	shp   io.ReadSeeker
+	shape Shape
+	num   int32
+	// filename   string
 	filelength int64
 
+	/* Note: not used
 	dbf             io.ReadSeeker
 	dbfFields       []Field
 	dbfNumRecords   int32
 	dbfHeaderLength int16
-	dbfRecordLength int16
+	dbfRecordLength int16*/
```

Comment:
It would be great if you could also write unit tests for reading DBF files and SHX files, as we will probably need to use it in the future.

Reply:
@mimoham24 Told me not to test any unused code, Instead he told me to comment out the unused code

Reply:
Ok, we'll create a new issue that implement unit tests for DBF and SHX files.